git_machete/cli.py

```diff
@@ -647,8 +661,13 @@ def launch(orig_args: List[str]) -> None:
                 current_branch = git.get_current_branch()
                 machete_client.expect_in_managed_branches(current_branch)
                 machete_client.retarget_github_pr(current_branch)
+            elif github_subcommand == "sync":
```

Comment:
Whoops, when I deleted my Github token:

```
 git machete github sync 
Checking for unmanaged branches...
No branches to delete
Checking for open GitHub PRs...
Could not determine current user name, please check your token.
```

This means that the flow is broken (not reaching the `delete_untracked` phase unless you've got a Github token... it should not behave like that, it should at most display the warning if the token is missing.